Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+
+def get_commit_tag() {
+  return sh(returnStdout: true, script: "git log -n 1 --pretty=format:'%h'").trim()
+}
+
+node {
+  stage('Prep') {
+    checkout scm
+    tag = get_commit_tag()
+  }
+
+  stage ('Buld Image'){
+    python_app = docker.build('mdnwebdocs/developer-portal-app:${tag}', '-f ./dockerfiles/Dockerfile.app .')
+    node_app = docker.build('mdnwebdocs/developer-portal-static:${tag}', '-f ./dockerfiles/Dockerfile.static .')
+  }
+
+  stage('Push Image') {
+    printlin("Pushing images")
+    python_app.push()
```

Comment:
Oh, it looks like you're not using the `tag` now, so the `.push()` must do that be default? So maybe we no longer need the `get_commit_tag` function?

Reply:
We can be explicit here but when you don't specify anything it pushes the image that we built, in this case its `docker.build('mdnwebdocs/developer-portal-app:${tag}', '-f ./dockerfiles/Dockerfile.app .')`

Which means it pushes the tagged name and the pushes the `latest` tag